tests/benchmarks/latency_measure/src/thread_switch_yield.c

```diff
@@ -79,8 +74,7 @@ void thread_switch_yield(void)
 	}
 
 	/* get the number of cycles it took to do the test */
-	TIMING_INFO_PRE_READ();
-	timestamp_end = TIMING_INFO_OS_GET_TIME();
+	timestamp_end = timing_counter_get();
```

Comment:
the while loop may take long enough so the underlying cycles' counter infrastructure, which supplies measurements to timing_counter_get() may overflow and the timestamp_end - timestamp_start won't correspond to the elapsed time.